pallets/capacity/src/lib.rs

```diff
@@ -1109,13 +1117,8 @@ impl<T: Config> StakingRewardsProvider<T> for Pallet<T> {
 
 	// Calculate the size of the reward pool for the current era, based on current staked token
 	// and the other determined factors of the current economic model
-	fn reward_pool_size(total_staked: BalanceOf<T>) -> BalanceOf<T> {
-		if total_staked.is_zero() {
-			return BalanceOf::<T>::zero()
-		}
-
-		// For now reward pool size is set to 10% of total staked token
-		total_staked.checked_div(&BalanceOf::<T>::from(10u8)).unwrap_or_default()
+	fn reward_pool_size(_total_staked: BalanceOf<T>) -> BalanceOf<T> {
+		T::RewardPoolEachEra::get()
```

Comment:
Question: Since this is a constant and it seems like based on the nature of `RewardPool` it would change and evolve. What might be the consequences of changing this in future? Since this is constant we are not able to dictate the exact time of the change so what happens if it changes between  two eras?

Reply:
I got the impression from some comments elsewhere that having this be a constant is a temporary solution just to get the logic of how to compute rewards into the codebase. Not sure if that's actually the case or if I'm misremembering something I heard, but either way might be good to elaborate in a comment.

Reply:
There were some placeholders before, yes, that was when it had not been determined how it would be calculated.  But, officially now, the Reward Pool size is a fixed constant for the purposes of setting inflation rates based on calculations and estimates  (referring you back to the spreadsheet, DM me if you need the link again).  If the reward pool size changes between two eras, it won't matter because that historical data is stored in `StakingRewardPool`, and that will be used to calculate the reward for that era.  Now if the formula changes... that's a whole other ball of worms [sic].  We'll just have to cross that bridge when if and when we get to it.